### ISSUE TITLE
Could not find com.google.firebase:firebase-core:17.0.0.

### ISSUE THREAD

**Issue by MartinSandstrom** [urbanairship/react-native-module#134]
Version: 2.0.0 on Android

The module installs com.google.firebase:firebase-core:17.0.0. This causes this error:

```
What went wrong:
Could not resolve all files for configuration ':app:debugCompileClasspath'.
> Could not find com.google.firebase:firebase-core:17.0.0.
  Required by:
      project :app > project :urbanairship-react-native
```

According to [firebase](https://firebase.google.com/support/release-notes/android), the latest version of firebase-core is 16.0.1. 

When i change the dependency to be 16.0.0, everything works.

Am i missing something?

**Comment by BrianBatchelder** [comment]
Looking at this now.

**Comment by BrianBatchelder** [comment]
I have reproduced this problem. A fix will be released later today.

**Comment by rlepinski** [comment]
Fixed in 2.0.1.

**Comment by BrianBatchelder** [comment]
@MartinSandstrom : This is fixed in [version 2.0.1](https://www.npmjs.com/package/urbanairship-react-native/v/2.0.1).

**Comment by MartinSandstrom** [comment]
Thanks for the quick help! 👍
